pages/common/sbt.md

```diff
@@ -0,0 +1,20 @@
+# sbt
+
+> Build tool for Scala and Java projects.
+> More information: <https://www.scala-sbt.org/1.0/docs/index.html>.
+
+- Start the SBT interactive shell (REPL):
+
+`sbt`
+
+- Use the specified version of sbt:
+
+`sbt -sbt-version {{version}}`
+
+- Use specified jar as the sbt launcher:
```

Comment:
Oops! Looks like I missed one. Perhaps this will read better?
```suggestion
- Use a specific jar file as the sbt launcher:
```

Reply:
Also agree, thank you again 👍